CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Version changelog
 
+## [Release] Release v1.52.0
+
+### New Features and Improvements
+
+ * Add support for filters in `databricks_clusters` data source ([#4014](https://github.com/databricks/terraform-provider-databricks/pull/4014)).
+ * Added `no_wait` option for clusters to skip waiting to start on cluster creation ([#3953](https://github.com/databricks/terraform-provider-databricks/pull/3953)).
+ * Introduced Plugin Framework ([#3920](https://github.com/databricks/terraform-provider-databricks/pull/3920)).
+ * Library plugin framework migration ([#3979](https://github.com/databricks/terraform-provider-databricks/pull/3979)).
```

Comment:
Imho it should be internal changes

Reply:
I have moved the library to internal but I think since plugin framework introduction is big and something we want to highlight for future external contributions as well (i.e. new resources to use plugin framework). I think having this in feature is okay.